octoprint_mrbeam/templates/hard_refresh_overlay.jinja2

```diff
@@ -19,7 +29,7 @@
         </div>
     </div>
     <div class="modal-footer">
-        <button class="btn btn-primary" id="#hard_refresh_confirmed" data-dismiss="modal"
-                aria-hidden="true" data-bind="click: $root.setUserHardRefreshed">{{ _('Yes, I will reload the page') }}</button>
+        <button class="btn btn-primary" id="hard_refresh_close_button" data-dismiss="modal"
+                aria-hidden="true" data-bind="click: $root.setUserHardRefreshed">{{ _('Close window') }}</button>
```

Comment:
```suggestion
        <button class="btn btn-primary" data-dismiss="modal"
                aria-hidden="true" data-bind="click: $root.setUserHardRefreshed">{{ _('Close window') }}</button>
```

Reply:
I had to change the button txt but this was removed, but it will be used full for automated testing, right? So we can click this button

Reply:
well for CyPress we will be adding custom attributes to handle the selection

Reply:
ok I will remove it then